app/views/kudos/show.html.erb

```diff
@@ -31,8 +31,10 @@
   <hr class="my-4">
 
   <div class="d-grid gap-1 d-md-flex mx-left">
-    <%= link_to 'Edit This Kudo...', edit_kudo_path(@kudo), class:"btn btn-warning" %>
-    <%= button_to 'Destroy This Kudo!', @kudo, method: :delete, :onclick => "return confirm('Are you sure you want to delete this Kudo?')", class:"btn btn-danger" %>
+    <% if @kudo.giver_of_kudo.email == current_employee.email %>
```

Comment:
Same thing as in index view - check if you can compare 
```suggestion
    <% if @kudo.giver_of_kudo == current_employee %>
```

Reply:
Done!